### PR TITLE
improve renovate

### PR DESCRIPTION
**Pull request by MarshallAsch**
- chore: move renovate file to github directory
- chore: group some of the dependancy updates together

Closes: # \
Fixes: # \
Related: # 

---

**Describe the pull request:**

[insert text here]

---

Checklist:

- [ ] Run `npm run lint` and ensure linter passes
- [ ] Run `npm run test` and ensure tests pass
- [ ] Verify that the changes work as expected on Discord
- [ ] Verify that the changes work as expected when run using docker
- [ ] Update documentation / not applicable
- [ ] Update changelog / not applicable

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "matchPackagePatterns": ["jest"],
+      "groupName": "jest",
+      "schedule": ["on the first day of the month"]
+    },
+    {
+      "matchPackagePatterns": ["^express"],
+      "matchPackageNames": ["cookie-parser", "helmet"],
+      "groupName": "express",
+      "schedule": ["on the first day of the month"]
+    }
```
Comment (SoorajModi) [inline .github/renovate.json:26]: For the rest of our dependencies, can we create another group like so:
```
    {
      "matchPackagePatterns": [
        "*"
      ],
      "groupName": "all dependencies",
      "groupSlug": "all",
       "schedule": ["on the first day of the month"]
    }
```
We can keep other groups for more sensitive dependencies

**Comment by SoorajModi** [comment]
We could also groups for:
- Discord.js 
- Mongoose
- Dev dependencies

File: .github/renovate.json (hunk 1 above)
Comment (MarshallAsch) [reply .github/renovate.json:26]: I dont think we can, there are quite a few of the other dependancies that require manual steps and cannot be bumped together.

Comment (SoorajModi) [reply .github/renovate.json:26]: All good then, worth a shot, ill approve

**Review by SoorajModi — APPROVED** [review]

**Pull request merged**